client/landing/stepper/utils/get-quantity-from-storage-slug.ts

```diff
@@ -1,7 +1,4 @@
-import {
-	FEATURE_50GB_STORAGE_ADD_ON,
-	FEATURE_100GB_STORAGE_ADD_ON,
-} from '@automattic/calypso-products';
+import { AddOns } from '@automattic/data-stores';
```

Comment:
What do you think about increasing the specificity to `PlanAddOns` instead of `AddOns`?  Or are there in fact add-ons that apply beyond just plans?

Reply:
They are more like site add-ons. But no, i don't think increasing specificity here makes sense 🤷